en/docs/learn/enterprise-integration-patterns/messaging-channels/invalid-message-channel.md

```diff
@@ -25,70 +26,85 @@ Before digging into implementation details, let's take a look at the relationshi
-        </outSequence>
-     <faultSequence>
-          <log level="full">
-                    <property name="MESSAGE" value="Failure Message..."/>
-                    <property name="ERROR_CODE" expression="get-property('ERROR_CODE')"/>
-                    <property name="ERROR_MESSAGE" expression="get-property('ERROR_MESSAGE')"/>
-              </log>
-         <drop />
-       </faultSequence>
-        </target>
-    <publishWSDL uri="file:repository/samples/resources/proxy/sample_proxy_1.wsdl"/>
-    </proxy>
-</definitions>
-```
+    <a href="{{base_path}}/assets/attachments/learn/enterprise-integration-patterns/InvalidMessageChannel.zip">
+    <img src="{{base_path}}/assets/img/integrate/connectors/download-zip.png" width="200" alt="Download ZIP"></a>
 
-When the server receives a request and the endpoint referred through the ESB profile is unavailable, the server triggers an error message. The ESB profile diverts the response to the invalid message channel. 
+4. Import the artifacts to WSO2 MI.
 
-## Set up the sample scenario
+    Click **File** -> **Open Folder** -> Select the extracted ZIP file to import the downloaded ZIP file.
+
+5. Start the project in the WSO2 MI server.
+
+    For instructions, go to [Build and Run]("{{base_path}}/develop/deploy-artifacts/#build-and-run") Documentation.
+
+## Execute the sample
+
+1. Save the following sample request as `payload.xml` in your local file system.
 
-Pass the following message to the ESB profile. You can use SoapUI or the ESB profile's Try It tool. To invoke the Try It tool, log into the Management Console of the ESB profile, navigate to the Services menu under the Main menu and select the List sub menu. Then select InvalidMessageChannelProxy and pass the following request to the ESB profile. 
+    ```xml
+    <soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:ser="http://services.samples" xmlns:xsd="http://services.samples/xsd">
+    <soapenv:Header/>
+    <soapenv:Body>
+        <ser:getQuote>
+            <ser:request>
+                <xsd:symbol>IBM</xsd:symbol>
+            </ser:request>
+        </ser:getQuote>
+    </soapenv:Body>
+    </soapenv:Envelope>
+    ```
 
-<soap:Envelope xmlns:soap="http://www.w3.org/2003/05/soap-envelope" xmlns:ser="http://services.samples" xmlns:xsd="http://services.samples/xsd">
-   <soap:Header/>
-   <soap:Body>
-      <ser:getQuote>
-         <!--Optional:-->
-         <ser:request>
-            <!--Optional:-->
-            <xsd:symbol>foo</xsd:symbol>
-         </ser:request>
-      </ser:getQuote>
-   </soap:Body>
-</soap:Envelope>
+2. Open a terminal, navigate to the location of your `payload.xml` file, and execute the following command. This posts a simple XML request to the Proxy service.
 
-### How the implementation works
+    ```bash
+    curl -L -H 'SOAPAction: urn:getQuote' -H 'Content-Type: text/xml' -d @payload.xml 'http://localhost:8290/services/InvalidMessageChannelProxy'
```

Comment:
Let's give how to invoke this using soapUI as proxy services are intended to use for soap.